config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -885,7 +885,7 @@ periodics:
   extra_refs:
   - org: kubernetes-sigs
     repo: cluster-api-provider-azure
-    base_ref: release-1.7
+    base_ref: release-1.8
```

Comment:
No test run since this change:

- https://testgrid.k8s.io/provider-azure-cloud-provider-azure#cloud-provider-azure-conformance-serial-vmss-capz

Reminder to check back later!